app/models/compliance/ObligationIdentification.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.compliance
+
+import play.api.libs.json.{Json, OFormat}
+
+case class ObligationIdentification(
+                                   incomeSourceType: Option[String],
```

Comment:
The incomeSourceType has enum as per the schema and, under a given condition it becomes mandatory. Do we not consider these. 
"incomeSourceType":{
                        "type":"string",
                        "enum":[
                           "ITSA",
                           "ITSB",
                           "ITSP",
                           "ITSF"
                        ],
                        "description":"Mandatory if   ReferenceType  is an IncomeSourceID"
                     },

Reply:
Reference type will never be IncomeSourceID until the service is made available under more regimes. So we don't consider this necessary (if we get the field, then fine - if not, we don't care)